### ISSUE TITLE
csharpier does not support .net 7

### ISSUE THREAD

**Issue by tanukihee** [belav/csharpier#767]
I successfully installed csharpier 0.20.0 but it can not work at .net 7.

```PowerShell
PS C:\Users\listl> dotnet --version
7.0.100

PS C:\Users\listl> dotnet csharpier
You must install or update .NET to run this application.

App: C:\Users\listl\.dotnet\tools\dotnet-csharpier.exe
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '6.0.0' (x64)
.NET location: C:\Users\listl\scoop\apps\dotnet-sdk\current

The following frameworks were found:
  7.0.0 at [C:\Users\listl\scoop\apps\dotnet-sdk\current\shared\Microsoft.NETCore.App]

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=6.0.0&arch=x64&rid=win10-x64
```

**Comment by belav** [comment]
I have this resolved in #756 which isn't released yet. The new version will be out sometime tomorrow.

**Comment by tanukihee** [comment]
Thank you. I found it reported at #752. I'll close it.
